Stripe/PublicHeaders/STPBlocks.h

```diff
@@ -47,3 +48,12 @@ typedef void (^STPErrorBlock)(NSError * __nullable error);
  *  @param error The error returned from the response, or nil in one occurs. @see StripeError.h for possible values.
  */
 typedef void (^STPTokenCompletionBlock)(STPToken * __nullable token, NSError * __nullable error);
+
+/**
+ *  A callback to be run with a validation result and shipping methods for a 
+ *  shipping address.
+ *
+ *  @param shippingValidationError If the shipping address is invalid, an error describing the issue with the address. Will be nil if the address is valid.
```

Comment:
I feel like we should make it easier for people to say "invalid shipping address", rather than having to initialize a (potentially localized) NSError object and call this callback with it. Two ideas:
- add a category on `NSError` to build one of these (might be a little weird with swift)
- add an additional `status` enum parameter to this method, a la `typedef void (^STPShippingMethodsCompletionBlock)(STPShippingStatus status, NSError * __nullable networkError, NSArray<PKShippingMethod *>* __nonnull shippingMethods);`